modules/raft/src/main/java/org/apache/ignite/internal/raft/storage/impl/DefaultLogStorageFactory.java

```diff
@@ -360,6 +369,49 @@ private static ColumnFamilyOptions createColumnFamilyOptions() {
         return opts;
     }
 
+    @Override
+    public Set<String> raftNodeStorageIdsOnDisk(GroupIdFastForward fastForward) {
+        Set<String> groupIdsForStorage = new HashSet<>();
+
+        byte[] upperBound = incrementPrefix(STORAGE_CREATED_META_PREFIX);
+
+        try (
+                Slice upperBoundSlice = new Slice(upperBound);
+                ReadOptions readOptions = new ReadOptions().setIterateUpperBound(upperBoundSlice);
+                RocksIterator iterator = db.newIterator(metaHandle, readOptions)
+        ) {
+            iterator.seek(STORAGE_CREATED_META_PREFIX);
+
+            while (iterator.isValid()) {
```

Comment:
If a new group appears during the method execution (for example, in parallel), then we won't see it like that? It won't ruin anything for us?

Reply:
I added a comment explaining that this method should only be called when groups cannot appear/be destroyed concurrently